src/orquestra/sdk/_ray/_dag.py

```diff
@@ -533,6 +533,35 @@ def get_workflow_run_status(self, workflow_run_id: WorkflowRunId) -> WorkflowRun
             message=message,
         )
 
+        if model.status.state.is_completed():
+            model = self._normalize_endtimes(model)
+
+        return model
+
+    @staticmethod
+    def _normalize_endtimes(model: WorkflowRun) -> WorkflowRun:
+        """Set the current time as end_time for tasks and workflows that don't have one.
+
+        Ray doesn't provide an end time for terminated tasks and workflows.
+        This brought some issues on Workflow Driver, we so fill up the missing status
+        fields with the current datetime for all terminated tasks and workflow.
+        """
+        # TODO use SDK's dates module:
+        # https://zapatacomputing.atlassian.net/browse/ORQSDK-943
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** ‏This comment doesn't make sense in this context.

Reply:
Does ORQSDK-943 still make sense?